Makefile

```diff
@@ -77,11 +77,16 @@ dep-update:
 # cloud targets - building images and deploying
 
 docker:
-	@echo Building Docker Image...
 	@mkdir -p _output/docker
 	@cp -r deploy/docker/* _output/docker
 	@cp ${GOPATH}/bin/sws _output/docker
-	@if [ ! -d "_output/docker/npm" ]; then npm --prefix _output/docker/npm -g install swsui; fi
+	@if [ ! -d "_output/docker/console" ]; then \
+		echo "Downloading console..." ; \
+		mkdir _output/docker/console ; \
+		curl $$(npm view swsui dist.tarball) \
```

Comment:
I think that at some point we might want to define a specific minor version to use to avoid mixing incompatible versions or even having old sws versions to use a know-to-work version.
e.g. `npm view swsui@0.2 dist.tarball --json | jq .[-1]`

Reply:
@josejulio  is there npm syntax for "give me the latest" ? If so, we can parameterize this in the Makefile:

`npm view swsui@${CONSOLE_VERSION} dist.tarbal ....`

with the default of CONSOLE_VERSION being whatever syntax means "latest".

The idea is that for right now at least we want this to build with the latest UI code, not any particular version (since we are moving fast, these versions are going to go out of date quick).

Reply:
ah sweet, answered my own question:

npm view swsui@latest

I'll tweek the Makefile to accept version.

Reply:
see PR #63